dbschema/seeds/017.partnerships.edgeql

```diff
@@ -0,0 +1,83 @@
+with
+partnershipsJson := to_json($$[
+  {
+      "partner": "Rohan Linguistics",
+      "project": "South Downs",
+      "mouStatus": "NotAttached",
+      "financialReportingType": "Funded",
+      "agreementStatus": "NotAttached",
+      "primary": true,
+      "types": ["Funding"]
+  },
+  {
+      "partner": "Ered Luin Translation Syndicate",
+      "project": "Emyn Muil",
```

Comment:
Partnerships are typically thought about by `project` first. Would you mind putting those first here?
Then `partner`, then `types`.